File: spec/code_samples/Node_unirest/auth/post.js
var unirest = require("unirest");

var req = unirest("POST", "https://analyzemail.com/api/v1/auth");

req.headers({
  "content-type": "application/json"
});

req.type("json");
req.send({
  "email": "<ADD STRING VALUE>",
  "password": "<ADD STRING VALUE>"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: spec/code_samples/Go_native/auth/post.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://example.com/api/v1/auth"

	payload := strings.NewReader("{\"email\":\"<ADD STRING VALUE>\",\"password\":\"<ADD STRING VALUE>\"}")

	req, _ := http.NewRequest("POST", url, payload)

	req.Header.Add("content-type", "application/json")
	req.Header.Add("authorization", "Bearer REPLACE_BEARER_TOKEN")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}